docs/koji.md

```diff
@@ -157,7 +159,10 @@ Example type-specific content generator metadata in context:
           "latest",
           "1.0.0-1",
           "1.0.0"
-        ]
+        ],
+        "digests": {
+          "application/vnd.docker.distribution.manifest.v2+json": "sha256:123def"
```

Comment:
To make things really clear, we could also add an example entry for manifest.v1, both here and in 'repositories' 12 lines above.

Reply:
Added an example with `v1` manifest (although the code would always prefer either v1 or v2 IIRC)